tox.ini

```diff
@@ -30,5 +30,11 @@ flake8-ignore =
     E501
     W503
     C901
+    py3status/modules/*.py W605 W504 E999
+filterwarnings =
+    # For Python 2 flake8
+    ignore:.*You passed a bytestring as `filenames`:DeprecationWarning:flake8.options.config
+    # For Python 3.7 flake8
+    ignore:Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated:DeprecationWarning:
```

Comment:
>, and in 3.8 it will stop working

Here's a diff to fix this now (only one) for future `python 3.8` issue. The rest would be from `python-udev`. 

```diff
diff --git a/py3status/storage.py b/py3status/storage.py
index 00641abf..2d523eb7 100644
--- a/py3status/storage.py
+++ b/py3status/storage.py
@@ -2,7 +2,11 @@ from __future__ import with_statement

 import os

-from collections import Iterable, Mapping
+try:
+    from collections.abc import Iterable  # python3
+except ImportError:
+    from collections import Iterable, Mapping  # python2
+
 from pickle import dump, load
 from tempfile import NamedTemporaryFile
 from time import time

```

Reply:
Poor comments signaling where it start/stop. It does not work anymore for python3.8 above, but  will work for python3.7 and below.